doc/cygnus-ngsi/flume_extensions_catalogue/last_data_function.md

```diff
@@ -0,0 +1,161 @@
+# Last Data functionality.
```

Comment:
Nice and comprehensive description of the functionality :)

However, let's try to avoid having a "orphan" documention. In particular:

* Include a link to it in the corresponding .md indexes files
* Include it in mkdocs.yml

Reply:
The postgresql sink document references this new document. Do you thing it should be included in any other document?

- [ ] mkdocs Fixed in [5a1e3dd](https://github.com/telefonicaid/fiware-cygnus/pull/1934/commits/5a1e3ddd765e6fc1e1b7e85efd5d80d98ea10cea)

Reply:
> The postgresql sink document references this new document. Do you thing it should be included in any other document?

You are right (I missed that in my first review :) and I that plus mkdocs.yml is enough.

NTC from my point of view